maintainers/maintainer-list.nix

```diff
@@ -3691,6 +3691,12 @@
     githubId = 362833;
     name = "Hongchang Wu";
   };
+  holgerpeters = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add holgerpeters``.